charts/featureform/values.yaml

```diff
@@ -104,6 +104,9 @@ psql:
   # If true, automatically creates a secret stored in templates/secrets/postgres.yaml.
   # If false, does not apply the template. The template can be created manually
   createSecret: false
+  migration:
+    image:
+      name: db-migration-up
```

Comment:
think you need to update the image github actions as well to include the new dockerfile.

Reply:
also look at `minikube_install_featureform` in the `makefile` as well.